display_list/display_list_complexity_metal.cc

```diff
@@ -35,15 +35,18 @@ DisplayListMetalComplexityCalculator::MetalHelper::BatchedComplexity() {
     // saveLayer seems to have two trends; if the count is < 200,
     // then the individual cost of a saveLayer is higher than if
     // the count is > 200.
-    if (save_layer_count_ > 200) {
-      // m = 1/5
-      // c = 1
-      save_layer_complexity = (save_layer_count_ + 5) * 40000;
-    } else {
-      // m = 1/2
-      // c = 1
-      save_layer_complexity = (save_layer_count_ + 2) * 100000;
-    }
+    //
+    // However, the trend is strange and we should gather more data to
+    // get a better idea of how to represent the trend. That being said, it's
+    // very unlikely we'll ever hit a DisplayList with 200+ saveLayer calls
+    // in it, so we will calculate based on the more reasonably anticipated
```

Comment:
... and if we do, it will easily end up exceeding our cache thresholds...

Reply:
Done